examples/platform_channel_swift/README.md

```diff
@@ -1,7 +1,6 @@
 # Example of calling platform services from Flutter
 
-This project demonstrates how to connect a Flutter app to platform
-specific services on iOS using Swift. The equivalent version of this
+This project demonstrates how to connect a Flutter app to platform-specific services on iOS using Swift. The equivalent version of this
```

Comment:
This line now looks too long (prefer 80 column lines)

Reply:
Okay will change it